theano/configdefaults.py

```diff
@@ -1267,6 +1280,7 @@ def default_blas_ldflags():
                               blas_info.get('library_dirs', [])])
                 res = try_blas_flag(flags)
                 if res:
+                    maybe_add_to_os_environ_pathlist('PATH', lib_path)
```

Comment:
I see that we will need here the function maybe_add_to_os_environ_pathlist for here. But I think we should call this method only on windows.